.nyc_output/039a84fc-6305-4919-b79b-b9f1c64c1724.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
none of the files in this folder should be committed

Reply:
My bad, fixed